modules/nf-core/biscuit/vcf2bed/main.nf

```diff
@@ -22,6 +22,7 @@ process BISCUIT_VCF2BED {
     def args2 = task.ext.args2 ?: ''
     def prefix = task.ext.prefix ?: "${meta.id}"
     """
+    set -o pipefail
```

Comment:
nf-core pipelines already have this, cf https://github.com/nf-core/tools/blob/master/nf_core/pipeline-template/nextflow.config#L175